db/system_keyspace.cc

```diff
@@ -2337,7 +2352,7 @@ std::vector<schema_ptr> system_keyspace::all_tables(const db::config& cfg) {
                     v3::cdc_local(),
                     raft(), raft_snapshots(), raft_snapshot_config(), group0_history(), discovery(),
                     topology(), cdc_generations_v3(), topology_requests(), service_levels_v2(), view_build_status_v2(),
-                    dicts(), view_building_coordinator_tasks(),
+                    dicts(), view_building_coordinator_tasks(), built_tablet_views(),
```

Comment:
Shouldn't this table (and the previous one) only by added if the schema feature is present?

Reply:
It doesn't matter here, we are already adding tables like `service_levels_v2`, `view_build_status_v2`, etc... without checking features.

More important is to add the table to raft snapshot only if the feature is enabled and this is already done.

Reply:
@avikivity `built_tablet_views()` is a local table (as all group0 tables). It will not cause schema mismatch issues.

As @Jadw1 wrote, we only need to make sure that we don't write to the new table through group0 until all nodes are upgraded. This should be handled by the cluster feature introduced in the first commit of this PR.